submissions/opeyemiPeter/challenge1-arrays/question3.js

```diff
@@ -0,0 +1,22 @@
+// Question 3
+
+function solution3(nums) {
```

Comment:
Welldone. This looks like a good solution. Also, it should work for all normal cases you can think of.
However, The time complexity is O(n^2). `nums.findIndex()` is an O(n) operation and it is nested inside a for loop.
You may run into time issues for questions with a lot of numbers.

Can you see if there's any way to reduce that to something lower say, O(nlogn) or even O(n)? Sorting and Hashtables are ideas you can try

Reply:
alright, i will try, to do that, is there a deadline for the correction?

Reply:
Not really. But it should probably be done before the next challenge, which is next week.